Library/Formula/racer.rb

```diff
@@ -0,0 +1,41 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
```

Comment:
`prefix` == `opt_prefix`, eventually. Once linked after install `/usr/local/opt/racer/racer` will really be located at `/usr/local/Cellar/racer/someversion/racer`. `opt_prefix` is just a symlink.

The trick would be getting text editors (or people) to set `/usr/local/opt/racer/racer` as the expected tool so it doesn't break across updates.

> Wouldn't libexec be more appropriate than opt_prefix?

I don't really mind either way. If it works and you prefer `libexec`, feel free. At that point `/usr/local/opt/racer/libexec/racer` will => `/usr/local/Cellar/racer/someversion/libexec/racer`.